### PR TITLE
pstoedit: add page

### LINKED ISSUE #7909: page request: pstoedit
http://www.pstoedit.net/

### PR DESCRIPTION
**Pull request by RedDeliciousApples**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
4.00

closes #7909

### CODE REVIEW

File: pages/linux/pstoedit.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# pstoedit
+
+> Convert PDF files into various image formats.
+> More information: <http://www.pstoedit.net/>.
+
+- Convert a PDF page to PNG or JPEG [f]ormat:
```
Comment (kbdharun) [inline pages/linux/pstoedit.md:6]:
```suggestion
- Convert a PDF page to PNG or JPEG format:
```
The long option doesn't seem to be present in the [manual](http://www.calvina.de/pstoedit/pstoedit.htm)

File: pages/linux/pstoedit.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`pstoedit -page {{page_number}}  -f magick {{filename}}  page.png|page.jpg`
```
Comment (kbdharun) [inline pages/linux/pstoedit.md:8]:
```suggestion
`pstoedit -page {{page_number}}  -f magick {{path/to/file.pdf}}  {{page.png|page.jpg]}}`
```

File: pages/linux/pstoedit.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Convert multiple PDF pages to numbered images:
+
+`pstoedit -f magick {{filename}} page%d.png|page%d.jpg`
```
Comment (kbdharun) [inline pages/linux/pstoedit.md:12]:
```suggestion
`pstoedit -f magick {{path/to/file}} {{page%d.png|page%d.jpg}}`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

I am not sure if `magick` format is supported. I couldn't test the official packages as the site's download link didn't work. I downloaded `pstoedit` from Fedora repos and this is what I got in the output:

[out.txt](https://github.com/tldr-pages/tldr/files/12880374/out.txt)

@pixelcmtd Can you confirm if this flag works?

File: pages/linux/pstoedit.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# pstoedit
+
+> Convert PDF files into various image formats.
+> More information: <http://www.pstoedit.net/>.
```
Comment (gutjuri) [inline pages/linux/pstoedit.md:4]:
```suggestion
> More information: <http://www.pstoedit.net>.
```

File: pages/linux/pstoedit.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+- Convert a PDF page to PNG or JPEG format:
+
+`pstoedit -page {{page_number}}  -f magick {{path/to/file.pdf}}  {{page.png|page.jpg]}}`
```
Comment (gutjuri) [inline pages/linux/pstoedit.md:8]:
```suggestion
`pstoedit -page {{page_number}} -f magick {{path/to/file.pdf}} {{page.png|page.jpg]}}`
```

**Review by gutjuri — CHANGES_REQUESTED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**